templates/etc_my.cnf-centos.j2

```diff
@@ -70,7 +70,7 @@ log_error = /var/log/mysqld.log
 {% if mysql_version_major|int < 8 %}
 log_warnings = 2
 {% else %}
-log_error_verbosity = 2
+log_error_verbosity = 1
```

Comment:
If warnings are not going to be printed anymore, would that affect this task?

https://github.com/artefactual-labs/ansible-percona/blob/17e6bc50295ebfb741a4530e8631bec8a83d7217/tasks/secure.yml#L13-L15

I can't remember if the temporary password is printed as a warning.

Reply:
@mamedin tested in all the supported environments and confirmed this doesn't break.

Reply:
I just tested in Ubuntu, Rocky8, Rocky9 and Almalinux and it works fine. The temporary password